packages/workspace/src/executors/run-script/run-script.impl.ts

```diff
@@ -2,9 +2,11 @@ import { execSync } from 'child_process';
 import { getPackageManagerCommand } from '@nrwl/devkit';
 import type { ExecutorContext } from '@nrwl/devkit';
 import * as path from 'path';
+import { env as appendLocalEnv } from 'npm-run-path';
```

Comment:
What is the reasoning behind adding the appendLocalEnv stuff? It doesn't seem relevant for the color functionality. In @nrwl/workspace:run-commands we use it so that a command like `jest` works without it being installed globally IIRC.